incubator/hnc/internal/validators/hncconfig_test.go

```diff
@@ -244,15 +253,15 @@ func TestPropagateConflict(t *testing.T) {
 	}
 }
 
-// fakeGVKValidator implements gvkValidator. Any kind that are in the slice are denied; anything else
+// fakeGVKValidator implements grValidator. Any kind that are in the slice are denied; anything else
```

Comment:
Nit: Change name to `fakeGRValidator`? Although it's not really a validator anymore - it's more like a translator. Maybe the interface name should be changed to `grTranslator` or `grLookup` or something like that (here and in the real code)? Before it really just checked if something existed to "validate" made sense, although even then (I now realize in retrospect) that name's a bit confusing since this entire package is called "validators."